src/SapiEmitter.php

```diff
@@ -109,8 +109,17 @@ private function emitBody(ResponseInterface $response): void
             $body->rewind();
         }
 
+        $level = ob_get_level();
         while (!$body->eof()) {
-            echo $body->read($this->bufferSize);
+            $output = $body->read($this->bufferSize);
+            if ($output === '') {
+                continue;
+            }
+            echo $output;
+            // flush the output buffer and send echoed messages to the browser
+            while (ob_get_level() > $level) {
+                ob_end_flush();
```

Comment:
[Seems](https://app.codecov.io/gh/yiisoft/yii-runner-http/pull/50/blob/src/SapiEmitter.php#L121), it's not covered by tests.